include/zcbor_encode.h

```diff
@@ -109,7 +109,8 @@ static inline bool zcbor_tstr_encode_ptr(zcbor_state_t *state, const char *ptr,
 	zcbor_tstr_encode_ptr(state, string, sizeof(string))
 
 /** Encode a tag. Must be called before encoding the value being tagged. */
-bool zcbor_tag_encode(zcbor_state_t *state, uint32_t tag);
+bool zcbor_tag_put(zcbor_state_t *state, uint32_t tag);
+bool zcbor_tag_encode(zcbor_state_t *state, uint32_t *tag);
```

Comment:
Would it be better to not break the API of this function but instead introduce the pointer argument variant in the new function?

Reply:
It will be the only exception to a rule that is followed by all the other APIs in the file :P. I think confusion for new users weighs heavily here, even if it changes for existing users. I can add a note in the docs. zcbor is still in "prerelease", so now is the time to do such changes IMO. Thoughts?

Reply:
I'm leaning on this clause here :D: https://semver.org/#spec-item-4

I'm hoping to clean zcbor up for a 1.0.0 release, and will abide by semver after that as well.

Reply:
Added a note in the header docs.